app/views/idv/otp_delivery_method/new.html.erb

```diff
@@ -24,29 +24,28 @@
     ) do |f| %>
 
   <fieldset class="margin-bottom-4 padding-0 border-none">
-    <label class="btn-border col-12 margin-bottom-1">
-      <div class="radio">
-        <%= radio_button_tag 'otp_delivery_preference', :sms, false, class: :otp_delivery_preference_sms %>
-        <span class="indicator mt-tiny"></span>
-        <span class="blue bold fs-20p">
-          <%= t('two_factor_authentication.otp_delivery_preference.sms') %>
-        </span>
-        <div class="regular gray-dark fs-10p mb-tiny">
-          <%= t('two_factor_authentication.two_factor_choice_options.sms_info_html') %>
-        </div>
-      </div>
+    <%= radio_button_tag(
+          'otp_delivery_preference',
+          :sms,
+          false,
+          class: 'otp_delivery_preference_sms usa-radio__input usa-radio__input--tile',
+        ) %>
+    <label class="usa-radio__label"><%= t('two_factor_authentication.otp_delivery_preference.sms') %>
+      <span class="usa-radio__label-description">
+        <%= t('two_factor_authentication.two_factor_choice_options.sms_info_html') %>
+      </span>
     </label>
-    <label class="btn-border col-12 margin-bottom-0">
-      <div class="radio">
-        <%= radio_button_tag 'otp_delivery_preference', :voice, false, class: :otp_delivery_preference_voice %>
-        <span class="indicator mt-tiny"></span>
-        <span class="blue bold fs-20p">
-          <%= t('two_factor_authentication.otp_delivery_preference.voice') %>
-        </span>
-        <div class="regular gray-dark fs-10p mb-tiny">
-          <%= t('two_factor_authentication.two_factor_choice_options.voice_info_html') %>
-        </div>
-      </div>
+
+    <%= radio_button_tag(
+          'otp_delivery_preference',
+          :voice,
+          false,
+          class: 'otp_delivery_preference_voice usa-radio__input usa-radio__input--tile'
+        ) %>
+    <label class="usa-radio__label"><%= t('two_factor_authentication.otp_delivery_preference.voice') %>
```

Comment:
WDYT of adding a newline?
```suggestion
    <label class="usa-radio__label">
      <%= t('two_factor_authentication.otp_delivery_preference.voice') %>
```

Reply:
This is one of the rare, unfortunate cases where whitespace matters in HTML. The newline collapses down to a single space, which throws off the layout.

I'd definitely agree it would hopefully be something we shouldn't need to worry about. I'm thinking about a few ways we could address this:

- Try to control whitespace behavior with CSS styling
- Use utilities like `label_tag`, etc. and avoid writing out the markup

Reply:
I think CSS styling would be ideal, or maybe we go with a comment approach

```suggestion
    <label class="usa-radio__label"><!-- can't have leading whitespace
      --><%= t('two_factor_authentication.otp_delivery_preference.voice') %>
```

Reply:
Follow-up at https://github.com/uswds/uswds/pull/4286. Still on the fence for how to handle it in the short-term.

Reply:
> @anniehirshman-gsa Good question. I would be fine with updating this in the design system. Might be worth a check with Chanan if it'd have any impact on her work on the upcoming partners site.

@aduth confirming that the larger text size (1.125 rem) is approved on the UX side for tiled radio buttons and checkboxes, and I've updated Figma. Figma also reflects the larger padding that would be great to update as well.

Reply:
Thanks @anniehirshman-gsa . I've got a [small conundrum to sort out with the whitespace improvements](https://github.com/uswds/uswds/pull/4286#discussion_r686981332) from this thread, but otherwise my plan will be to incorporate those changes as a temporary patch into the design system. I can make the tile font size changes around the same time. Since all this is non-ticketed refactoring, I might be a bit delayed in getting to it for prioritization's sake.

Reply:
If we want to merge this sooner rather than later, I think it might be nice to leave the commented version of the newline, and then we could leave in a "fix after USDWS update" in there too so we can find them easily if/when USWDS is handled to be more tolerant of whitespace